package.json

```diff
@@ -36,7 +36,9 @@
   },
   "dependencies": {
     "htmlparser2": "^3.8.3",
-    "lodash": "^3.9.3"
+    "lodash.foreach": "^4.1.0",
+    "lodash.compact": "^3.0.1",
```

Comment:
The modularized lodash packages' versions do not always correspond with the current lodash version, they are versioned independently.